apps/web/src/env.mjs

```diff
@@ -1,23 +1,37 @@
 import { createEnv } from "@t3-oss/env-nextjs";
 import { z } from "zod";
 
+import { optionalStringSchema } from "@blobscan/zod";
+
 export const env = createEnv({
   /**
    * Specify your server-side environment variables schema here. This way you can ensure the app isn't
    * built with invalid env vars.
    */
   server: {
-    DATABASE_URL: z.string().url(),
-    BEACON_NODE_ENDPOINT: z.string().url().default("http://localhost:3500"),
+    DATABASE_URL: optionalStringSchema(
+      z.string().url(),
+      "postgresql://blobscan:s3cr3t@localhost:5432/blobscan_dev?schema=public"
+    ),
+    BEACON_NODE_ENDPOINT: optionalStringSchema(
+      z.string().url(),
+      "http://localhost:3500"
+    ),
     NODE_ENV: z.enum(["development", "test", "production"]),
   },
   /**
    * Specify your client-side environment variables schema here.
    * For them to be exposed to the client, prefix them with `NEXT_PUBLIC_`.
    */
   client: {
-    NEXT_PUBLIC_EXPLORER_BASE_URL: z.string().url(),
-    NEXT_PUBLIC_BEACON_BASE_URL: z.string().url(),
+    NEXT_PUBLIC_EXPLORER_BASE_URL: optionalStringSchema(
```

Comment:
I would not make these variables optional neither. They are part of the minimum required for blobscan to work. We already provide default values in the .env.example file